ESCPOS_NET/Emitters/BaseCommandEmitter/CharacterCommands.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using ESCPOS_NET.Emitters.BaseCommandValues;
+using System;
 
 namespace ESCPOS_NET.Emitters
 {
     public abstract partial class BaseCommandEmitter : ICommandEmitter
     {
+        public int Columns { get; set; } = 42;
```

Comment:
I wasn't sure about the default for this, 42 columns is what the majority of printers I dealt with in Brazil had, but as an alternative, we could just not set any default value for the property, make it a getter-only property and force the constructor to receive the column count. However that would be a breaking change, so between having a random number as default and breaking existing code from others, I decided to stick with a default value. 

What do you think @lukevp ?